recipes/vmd-python/LICENSE

```diff
@@ -0,0 +1,135 @@
+UNIVERSITY OF ILLINOIS
```

Comment:
This file isn't necessary, and isn't even being used by the recipe; when you do `license_file: LICENSE` it pulls it from the source repo (which is better). You could just delete this file from the recipe.